src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/Common/Notifications/Brokers/SmtpEmailSenderBroker.cs

```diff
@@ -22,7 +22,7 @@ public class SmtpEmailSenderBroker(IOptions<SmtpEmailSenderSettings> smtpEmailSe
     /// <returns>A ValueTask&lt;bool&gt; representing the asynchronous operation's success status.</returns>
     public ValueTask<bool> SendAsync(EmailMessage emailMessage, CancellationToken cancellationToken = default)
     {
-        emailMessage.SenderEmailAddress ??= _smtpEmailSenderSettings.CredentialAddress;
+        emailMessage.SenderEmailAddress = _smtpEmailSenderSettings.CredentialAddress;
```

Comment:
Because if other user is sending an email the user should provide its own email credentials (like password). But now we do not have any business logic here that would enable users send personal emails. That's why all emails are sent by the system email address.